test/integration/Makefile

```diff
@@ -326,3 +326,6 @@ test_lookup_properties:
 
 args:
 	(cd targets/args && ./runme.sh $(TEST_FLAGS))
+
+test_callback_retry_task_name:
```

Comment:
Added a test/integration/targets/*/aliases file, so ansible-test will find the tests and run it from CI and locally.